src/BuildIntegration/Microsoft.NETCore.Native.Unix.props

```diff
@@ -40,6 +40,7 @@ See the LICENSE file in the project root for more information.
   <ItemGroup>
     <AdditionalNativeLibrary Include="$(IlcPath)/sdk/libSystem.Private.CoreLib.Native.a" />
     <AdditionalNativeLibrary Include="$(IlcPath)/framework/System.Native.a" />
+    <AdditionalNativeLibrary Include="$(IlcPath)/framework/libSystem.Globalization.Native.a" />
```

Comment:
Thanks for the pointer @janvorli. I'll update libSystem.Globalization.Native.a name